pkgs/development/python-modules/pytest-env/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pytest }:
+
+buildPythonPackage rec {
+  pname = "pytest-env";
+  version = "0.6.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hl0ln0cicdid4qjk7mv90lw9xkb0v71dlj7q7rn89vzxxm9b53y";
+  };
+
+  propagatedBuildInputs = [ pytest ];
```

Comment:
Don't propagate `pytest`, it's a special case. Plugins should only use it for build time of itself.